sklearn/feature_selection/from_model.py

```diff
@@ -111,3 +139,117 @@ def transform(self, X, threshold=None):
             return X[:, mask]
         else:
             raise ValueError("Invalid threshold: all features are discarded.")
+
+
+class SelectFromModel(BaseEstimator, SelectorMixin):
+    """Meta-transformer for selecting features based on importance weights.
+
+    Parameters
+    ----------
+    estimator : object
+        The base estimator from which the transformer is built.
+        This can be both a fitted (if ``prefit`` is set to True)
+        or a non-fitted estimator.
+
+    threshold : string, float, optional
+        The threshold value to use for feature selection. Features whose
+        importance is greater or equal are kept while the others are
+        discarded. If "median" (resp. "mean"), then the ``threshold`` value is
+        the median (resp. the mean) of the feature importances. A scaling
+        factor (e.g., "1.25*mean") may also be used. If None and if
```

Comment:
I don't see where an object attribute "threshold" is used. Also you should document the L1 special case.